src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx

```diff
@@ -627,19 +629,29 @@ If you need assistance with New Relic Products, you are in good hands with sever
     The New Relic [Community Forum](https://discuss.newrelic.com/) is 100% free and open to anyone with a New Relic account. The community is a place where many customer questions have already been asked and answered. Answers come from our community of experienced users, New Relic Support Engineers, and dozens of other Relics who help answer questions and solve problems. If you want to ask a question, check the community - if your question has not already been answered, members of the community can help.
   </Collapser>
 
-  <Collapser
-    id="github"
-    title="Github"
-  >
-    We want everyone to monitor their systems, and we're contributing our technology back to the open-source community to make that happen. You can find all our open source projects in our [Github repo](https://github.com/newrelic). We're committed to open standards, open-sourcing all of our instrumentation, and engaging engineers where they are, in the communities they already belong to. For more information on support for Open Source projects, visit our [Open Source Support Policy](https://docs.newrelic.com/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings#h2-open-source-support).
-  </Collapser>
-
-  <Collapser
-    id="nrdiag"
-    title="Diagnostic tools"
-  >
-    New Relic offers a [diagnostics utility](https://docs.newrelic.com/docs/using-new-relic/cross-product-functions/troubleshooting/new-relic-diagnostics) that can automatically detect common problems with New Relic agents. If Diagnostics detects a problem, it suggests troubleshooting steps. New Relic Diagnostics can also automatically attach troubleshooting data to a New Relic Support ticket. We have also made available [Troubleshooting Frameworks](https://discuss.newrelic.com/t/troubleshooting-frameworks/108787) that step users through common troubleshooting questions.
-  </Collapser>
+{' '}
+<Collapser id="github" title="Github">
```

Comment:
It looks like prettier might have mangled this file a bit. Would you be willing to undo these changes and redo the video change without formatting this file?

Reply:
Sure.. curious as to why, it was a very simple change and I just saved the file.